### ISSUE TITLE
Flyout menu aria expanded on hover

### ISSUE THREAD

**Issue by dfabian** [w3c/wai-website#628]
[Fly-out Menu](https://www.w3.org/WAI/tutorials/menus/flyout/#flyoutnavkbbtn) examples are a tad inconsistent here. The second example of `li.has-submenu a` does not trigger an aria-expanded state change on hover yet the third example does. Should it? I see no mention of a mouseenter triggering a click event here or anything else.

Edit: I do see this [#525](https://github.com/w3c/wai-tutorials/issues/525) however that is seemingly in conflict with other resources such as [this article](https://www.smashingmagazine.com/2017/11/building-accessible-menu-systems/). 

_"As pictured, whether that menu is open or closed should be communicated with aria-expanded. You should only change this state on click, not on focus. Users do not usually expect an explicit change of state on a mere focus event. In our navigation system, state doesn’t really change; it’s just a styling trick. Behaviorally, we can Tab through the navigation as if no such show/hide trickery were occurring."_

Thanks much.
